components/nexus-security/src/main/java/org/apache/shiro/nexus/NexusWebSecurityManager.java

```diff
@@ -28,6 +28,14 @@
 public class NexusWebSecurityManager
     extends DefaultWebSecurityManager
 {
+
+  /**
+   * Constructor which disables all RememberMe functionality.
+   */
+  public NexusWebSecurityManager() {
+    this.setRememberMeManager(null);
```

Comment:
to prevent always getting rememberMe=deleteMe cookie on every damn request,

hrm, well when I port this to NX3 I guess this may not apply.